scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScaladocParser.scala

```diff
@@ -186,7 +210,7 @@ object ScaladocParser {
     def cellChar = escape ~ AnyChar | !(nl | escape | tableSep) ~ AnyChar
     def row = (cellChar.rep.! ~ tableSpaceSep).rep(1)
     // non-standard but frequently used delimiter line, e.g.: +-----+-------+
-    def delimLine = hspaces0 ~ plusMinus
+    def delimLine = hspacesMin(indent) ~ plusMinus
```

Comment:
this is also not needed, a table doesn't have to be indented within a list, it looks like.